fit/curveNd_test.go

```diff
@@ -0,0 +1,189 @@
+// Copyright ©2017 The go-hep Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package fit_test
+
+import (
+	"bufio"
+	//"image/color"
```

Comment:
please remove.

Reply:
...not sure why that's commented out, it shouldn't be (not sure how it worked if it was commented out!)

Reply:
it would seem this is still there :)

Reply:
My remote branch is not receiving all of the changes on my local branch. I'll do some investigation and get back to you when its sorted.